Source/WebCore/html/HTMLTableElement.cpp

```diff
@@ -509,10 +509,10 @@ Ref<MutableStyleProperties> HTMLTableElement::createSharedCellStyle() const
         style->setProperty(CSSPropertyBorderColor, CSSPrimitiveValue::create(CSSValueInherit));
         break;
     case SolidBordersRowsOnly:
-        style->setProperty(CSSPropertyBorderTopWidth, CSSValueThin);
-        style->setProperty(CSSPropertyBorderBottomWidth, CSSValueThin);
-        style->setProperty(CSSPropertyBorderTopStyle, CSSValueSolid);
-        style->setProperty(CSSPropertyBorderBottomStyle, CSSValueSolid);
+        style->setProperty(CSSPropertyBorderTopWidth, CSSPrimitiveValue::create(0, CSSUnitType::CSS_PX));
+        style->setProperty(CSSPropertyBorderBottomWidth, CSSPrimitiveValue::create(0, CSSUnitType::CSS_PX));
+        style->setProperty(CSSPropertyBorderTopStyle, CSSPrimitiveValue::create(CSSValueNone));
+        style->setProperty(CSSPropertyBorderBottomStyle, CSSPrimitiveValue::create(CSSValueNone));
```

Comment:
Where in the spec is this defined? Can you provide a link?

I only found: https://html.spec.whatwg.org/#tables-2 but it doesn't quite match what you're defining here.

Reply:
Let me re-check.
https://html.spec.whatwg.org/multipage/rendering.html#tables-2:presentational-hints
And I will create a WPT test for it.
Thanks!